go/test/endtoend/tabletmanager/throttler_topo/throttler_test.go

```diff
@@ -358,6 +359,36 @@ func TestInitialThrottler(t *testing.T) {
 	})
 }
 
+func TestThrottleViaApplySchema(t *testing.T) {
+	defer cluster.PanicHandler(t)
+	t.Run("throttling via ApplySchema", func(t *testing.T) {
+		vtctlParams := &cluster.ApplySchemaParams{DDLStrategy: "online"}
+		_, err := clusterInstance.VtctldClientProcess.ApplySchemaWithOutput(
+			keyspaceName, "alter vitess_migration throttle all", *vtctlParams,
+		)
+		assert.NoError(t, err)
+	})
+	t.Run("validate keyspace configuration", func(t *testing.T) {
+		keyspace, err := clusterInstance.VtctldClientProcess.GetKeyspace(keyspaceName)
+		require.NoError(t, err)
+		require.NotNil(t, keyspace)
+		require.NotNil(t, keyspace.Keyspace.ThrottlerConfig)
+		require.NotEmpty(t, keyspace.Keyspace.ThrottlerConfig.ThrottledApps, "throttler config: %+v", keyspace.Keyspace.ThrottlerConfig)
+		appRule, ok := keyspace.Keyspace.ThrottlerConfig.ThrottledApps[throttlerapp.OnlineDDLName.String()]
+		require.True(t, ok, "throttled apps: %v", keyspace.Keyspace.ThrottlerConfig.ThrottledApps)
+		require.NotNil(t, appRule)
+		assert.Equal(t, throttlerapp.OnlineDDLName.String(), appRule.Name)
+		assert.EqualValues(t, 1.0, appRule.Ratio)
+	})
+	t.Run("unthrottling via ApplySchema", func(t *testing.T) {
```

Comment:
If we're going to test the unthrottling as well, then IMO it makes sense to validate the keyspace config afterwards too.

Reply:
Done.